misc/images/mz/mzbuild.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+name: mz
+pre-image:
+  - type: cargo-build
+    bin: [mz]
+    strip: false
+publish: false
```

Comment:
@benesch DockerHub returns a "permission denied" when attempting to publish this container. Is there a registry of allowed container names somewhere? I could not find any mention of this in i2

Hence the `publish: false` and the 20-minute compilation times (which I assume are because of the workspace crate?)

Reply:
Thank you.